### ISSUE TITLE
Remove c:geo wear from recommended apps

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#6515]
The mentioned Android wear app has been depublished as the author stopped maintaining it.
Very sad IMHO.

However we should remove it from our list of recommended apps for now.
I contacted the author, if we could takeover the code.

If someone does the needed changes: Please apply them to `release` branch.

**Comment by Lineflyer** [comment]
The author is fine if we take the code into our repo (in case someone on future is interested to maintain/evolve it). How would this work?
See here:
https://github.com/cjmoran/cgeo-wear/issues/20

**Comment by Bananeweizen** [comment]
The wear app is under Apache license, so legally this is easy. For the actual proceeding I assume we could just fork the complete repository as a separate repository under our cgeo account and refactor it before potentially moving it into our main repository. However, I have not yet investigated the code, so I'm not aware how well it fits our remaining app.

**Comment by Lineflyer** [comment]
Let me reopen this...could someone fork the code (repo mentioned above) to our project?
I came back to this topic as I bought a new smartwatch and recognized we not only removed the link to "helper app" but also removed the intent from the c:geo code....so although I still have the wear app I cannot use it.
Why not just test and publish it again as it is...it was working for me at that time.

**Comment by SammysHP** [comment]
We got a request on Twitter if we can add the "send to wear" feature again: https://twitter.com/f3lis_catus/status/977130306921816066

**Comment by kbudde** [comment]
Hi, 
please do not remove c:geo wear from the recommended apps.
It would be nice to have everything enabled again.

I've no real life experience with maintaining and developing android/wear apps. 
Nevertheless I've forked the projects now and going to hack on them. 
Currently I cannot promise any results but if I'm able to get it running perhaps I'm feeling lucky enough to continue...

**Comment by rsudev** [comment]
If you plan to publish a new version of the app, I guess you will have to use a new name, so we would need to adapt regardless, but that is a small and easy thing to do. So keep us posted!

**Comment by kbudde** [comment]
I've checked the code so far. It was not hard to get it working again.
The code is quite old and not really working with current android versions (location permission requests are missing; for wear app I would recommend using the ambient mode to reduce power consumption,..).
So some more work is required. Today I'm going to test the current solution outside.

Current architecture consists of a mobile app and a wear app. The mobile app is used by cgeo for sharing the cache details. The mobile app is sending the details to the wear app.

I would create a new github issue (implement wear app) and provide "later" a PR containing changes for cgeo:
- add a new wear companion app for cgeo
- add the necessary changes to send the cache details to the companion app from cgeo.
I would start from scratch. 
IMHO it would be a much cleaner solution than reusing the current app+wear app.

This issue #6557 should be closed as already everything for the wear app was removed.
